crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -761,9 +738,49 @@ async fn execute_operation(
 
     match child {
         Ok(child) => {
```

Comment:
The name `child` is misleading. I started to wonder what was the relationship with child devices before understanding that it's about a child process!

`child_process` would be clearer.